sbi/inference/base.py

```diff
@@ -179,6 +187,66 @@ def get_simulations(
 
         return theta, x, prior_masks
 
+    def append_simulations(
+        self,
+        theta: Tensor,
+        x: Tensor,
+        exclude_invalid_x: bool = False,
+        from_round: int = 0,
+        algorithm: str = "SNLE or SNRE",
```

Comment:
make this `Optional[str]=None` and handle the `None` below.

Reply:
Done!